### DISCUSSION TITLE
Running deadOS without local-Host only local path:out/index.html

### DISCUSSION THREAD

**Discussion by JackTelford** [DustinBrett/daedalOS#453] [Q&A]
Hey Dustin I have completed 99% of this integration but when I run yarn build I get my out folder and I run index.html server-less no local host I get a blank screen. Have you tried to run the out folder index.html only? This is my last step to my project so hoping you may have some insight to making it run without a local host. My apps I have made do not require any internet connection so this is offline. Thank you.!

**Comment by DustinBrett** [comment]
> Hey Dustin I have completed 99% of this integration but when I run yarn build I get my out folder and I run index.html server-less no local host I get a blank screen. Have you tried to run the out folder index.html only? This is my last step to my project so hoping you may have some insight to making it run without a local host. My apps I have made do not require any internet connection so this is offline. Thank you.!

It should work but it depends if you are serving the file in a way where it knows the relative location of supporting documents. Look at the DevTools Network/Console to see what the issue is. Probably something isn't loading. The root folder needs to be where everything is but if you open the index.html file from `file:///C:/x/out/index.html` then it will be relative to `C:/`. The same thing happens for me and it means you need to either serve the files on root or you need to use a web server to host files in a way where it treats the host directory as root.

![image](https://github.com/DustinBrett/daedalOS/assets/16656318/b780b35f-5147-4051-a478-8244796dff09)

**Comment by JackTelford** [comment]
Hey Dustin There are some scripts that want to use importScripts but since this is going to be a local only project how could I get them to import correctly?   <script src="./_next/static/chunks/Stable Diffusion.308a3d825ac1af8f.js" defer=""></script>

Side note when you run the index.html with the chunks needed it will load the site forever like a (do)loop. Endlessly.

**Reply by DustinBrett** [reply]
> Hey Dustin There are some scripts that want to use importScripts but since this is going to be a local only project how could I get them to import correctly? <script src="./_next/static/chunks/Stable Diffusion.308a3d825ac1af8f.js" defer=""></script>
> 
> Side note when you run the index.html with the chunks needed it will load the site forever like a (do)loop. Endlessly.

I don't have much advice on how to run the code without some kind of web server or via running it from a root drive. The looping is because things aren't loading correctly.

**Reply by JackTelford** [reply]
The fix to this is in the frame work chunk line 2209 on initialization of components. I have completed the app in a Iframe server-less and I have it working with initialization Directly in a Razor view pretty cool. Thank you for your help alone the way.

**Reply by DustinBrett** [reply]
Glad you got it working!
